### PR TITLE
Update trussels.md

### PR DESCRIPTION
**Pull request by KristinWegner**
Updated blurb about editing the Trussels' Guide - included a link to the GitHub Guide so people can learn how to propose changes directly

### CODE REVIEW

File: docs/trussels.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This is the `Trussels' Guide to Truss`, written by Trussels for Trussels. We sha
...
+> **Disclaimer for Trussels**: This is a living document and should be considered malleable and fallible. The Founders have not read this document in a long time, and certainly do not read every edit. It really is not an employee handbook or formal statement of policy. If you have any questions about this guide or want to know Truss’s policy on any of these issues, please contact Jen or another Founder. If something seems like it’s wrong, or missing, or whatever, please suggest a change or ask about it in #onboarding on Slack. You can also edit the guide directly (["Everyone's GitHub Guide to Updating the Trussels' Guide"] https://docs.google.com/document/d/1gHQehHRlgHh5MrOUmM6DMcTz6EFNQE1HcgNiE-whh8g/edit#heading=h.qzrl50vb7ejp). You can see when it was last updated in the version history.
```
Comment (ferlatte) [inline docs/trussels.md:12]: This is great. Two small changes requested: 1: can you clean up the the doc URL by removing everything after `/edit`, unless you mean for the link to go to a specific document heading (in which case just say so and it'll go through). The other request is I think this doc isn't public; we have a convention of putting a `:lock:` emoji in front of private links so external readers don't get confused about why the link doesn't work for them.

**Review by ferlatte — CHANGES_REQUESTED** [review]
Two small nits requested, thank you so much!

File: docs/trussels.md (hunk 1 above)
Comment (KristinWegner) [reply docs/trussels.md:12]: Oops- I made the doc public- the link should now work for any trussel.

**Comment by KristinWegner** [comment]
Oops- I made the doc public- the link should now work for any trussel.

**Review by KristinWegner — COMMENTED** [review]
I moved the document over to the Truss folder, so it's accessible (and editable) by anyone.

**Comment by KristinWegner** [comment]
I moved the document over to the Truss folder, so it's accessible (and editable) by anyone.

**Pull request merged**